### DISCUSSION TITLE
Enhancement: Wallpaper settings

### DISCUSSION THREAD

**Discussion by AuthorShin** [IceWhaleTech/CasaOS#191] [Ideas]
Hello
Adding wallpaper manager and setting to customize it.
Wallpaper selection sources:

- Static colors (from hex code)
- Textures
- Gradients
- Option to select the wallpaper from Bing daily pictures
- Remote URL
- Upload directly
- A folder that is assigned to wallpapers so any picture in that folder would be set as wallpaper, randomly or sequentially and how much time it would take to change between them can be an extra section in the settings as well.

**Comment by Lauren-ED209** [comment]
Make it elegant and customizable

**Reply by AuthorShin** [reply]
Indeed
Was thinking about calm background music but realized it was a bit overkill :) and also might be out of scope of this project.

**Reply by Protektor-Desura** [reply]
I would think if you start talking music that you need to be sure of the license of the music being used.

**Reply by AuthorShin** [reply]
We can use services like Soundcloud or Spotify or other free ones that let you use the musics somehow.

**Comment by alekssze** [comment]
I would also love ability to change Background Image
